uzduotis06/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace uzduotis06
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Išveskite pasirinkto skaičiaus daugybos lentelę.
+            int number = 8;
+            Console.WriteLine(number * 1);
+            Console.WriteLine(number * 2);
+            Console.WriteLine(number * 3);
+            Console.WriteLine(number * 4);
+            Console.WriteLine(number * 5);
+            Console.WriteLine(number * 6);
+            Console.WriteLine(number * 7);
+            Console.WriteLine(number * 8);
+            Console.WriteLine(number * 9);
+            Console.WriteLine(number * 10);
```

Comment:
Čia norėjosi, kad būtų išvesta pilna informacija (1 * 1 = 1; 1 * 2 = 2), bet gal tinkamai to neiškomunikavau :(

Reply:
Ačiū. Pataisiau su nauju commit.